vmdb/Gemfile

```diff
@@ -42,7 +42,13 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "default_value_for",              "~>2.0.3"
-gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+
+if RUBY_PLATFORM == 'java'
+  gem "puma"
+else
+  gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+end
```

Comment:
:+1: for puma.

Ran torquebox/JRuby in production. Issues with JRuby on the edge cases (e.g. jdbc adapter and supporting non standard sql database types). Enjoyed torquebox for the most part.

Going towards puma basically buys us better support for puma, passenger, and others that are more thread friendly. Feel unicorn pushes us towards the forking model.